index.bs

```diff
@@ -3681,11 +3681,44 @@ executes script. The <code>ScriptResultSuccess</code> variant is used in cases
 where the script completes normally and the <code>ScriptResultException</code>
 variant is used in cases where the script completes with a thrown exception.
 
+#### The script.Source type #### {#type-script-Source}
+
+[=Local end definition=]
+
+<pre class="cddl local-cddl">
+Source = (
+  realm: Realm,
+  ?context: BrowsingContext
+);
+</pre>
+
+The <code>Source</code> type represents a <code>script.Realm</code> that
```

Comment:
I don't know if it represents a realm, it represents the realm in which a script related event occured (or something like that).

Reply:
Maybe it's better now?